Cloud_Providers.md

```diff
@@ -0,0 +1,50 @@
+# Monitoring and Logging on Cloud Provider Platforms
+
+This document contains information about using the SAS Viya logging and monitoring components in cloud provider platforms (Azure, Amazon, and Google) that are supported by SAS Viya.
+
+The logging and monitoring applications provided in this repository for SAS Viya can be deployed and can run in any of the cloud provider's environments. Minimal customization is needed to deploy these applications. Because the applications provided for SAS Viya work the same way across cloud platforms, administrators can have a consistent experience when moving between cloud platforms. 
+
+The configurations for the monitoring and logging applications provided for SAS Viya are focused on detailed metrics and messages from the applications and services that make up SAS Viya. In some cases, the monitoring application provided by a cloud provider does not support the same level of detail. For administrators who are interested in metrics and log messages from SAS Viya services and components, the applications included in this repository provide the most comprehensive view of this detailed information. For administrators who are primarily interested in information from cluster resources, native applications might provide the level of logging and monitoring support that is needed.
+
+Although each supported cloud provider provides one or more native applications to display metrics and log messages, configuration and customization steps are required in order to integrate collected SAS Viya metric data and log messages with these applications. Samples are included in this repository to provide the steps required to display SAS Viya metrics and log messages on these applications. However, not all scenarios are currently supported.   
+
```

Comment:
Even saying "not all possible scenarios are supported" sounds incorrect to me. Just because we don't demonstrate how to do something doesn't mean it can't be accomplished or is not supported. When I read that, it seems that we know it doesn't work vs. there being a solution, just no sample that demonstrates it.

Reply:
Changed to "However, samples are not currently provided for all possible scenarios."

Reply:
I think the latest revisions address my concern.